game/scripts/npc/abilities/earthshaker_echo_slam.txt

```diff
@@ -52,13 +49,13 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "echo_slam_echo_damage"                           "110 130 150 275 400" //OAA
+        "echo_slam_echo_damage"                           "110 130 150 375 500" //OAA
         "LinkedSpecialBonus"                              "special_bonus_unique_earthshaker_2"
       }
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "echo_slam_initial_damage"                        "100"
+        "echo_slam_initial_damage"                        "100 200 300 600 900" //OAA
```

Comment:
At OAA levels I would put `500 800` to match what was already meant to be at max level.